src/display-buffer.coffee

```diff
@@ -465,6 +465,10 @@ class DisplayBuffer extends Model
 
     folds
 
+  # Returns all the folds that contain the given row range.
+  foldsContainingBufferRowRange: (startRow, endRow) ->
```

Comment:
I think this should be named `foldsIntersectingBufferRowRange`, based on the query.